sdcm/sct_runner.py

```diff
@@ -52,7 +52,7 @@
 from sdcm.utils.azure_region import AzureOsState, AzureRegion, region_name_to_location
 from sdcm.utils.get_username import get_username
```

Comment:
Linter fails because this import is unused. Need to delete it.

Reply:
@vponomaryov - done